apps/showcase/src/components/training/save-code-dialog/save-code-dialog.template.html

```diff
@@ -0,0 +1,10 @@
+<div class="modal-header">
+  <h2 class="modal-title">Code modifications detected</h2>
```

Comment:
```suggestion
  <h2 class="modal-title">Training in progress detected</h2>
```

Reply:
I personally prefer the original sentence `Code modifications detected` ...